### PR TITLE
Get input from form and update chart on submit

### PR DESCRIPTION
**Pull request by leeningyap**

### CODE REVIEW

File: amortisation-calculator/src/components/AmortisationChart.js [hunk 1]
```diff
@@ -1,34 +1,23 @@
...
-    //       'rgba(153, 102, 255, 1)',
-    //       'rgba(255, 159, 64, 1)'
-    //     ],
-    //     borderWidth: 1
-    //   }
-    // ]
+    labels: Array.from(Array(Number(loan.term)).keys()),
+    datasets: [
+      {
+        label: '# of Votes',
```
Comment (achoi0330) [inline amortisation-calculator/src/components/AmortisationChart.js:10]: Label should be updated?

File: amortisation-calculator/src/App.js [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
+    });
+  };
+
   return (
     <div className='App'>
       <h1>Amortisation Calculator</h1>
-      <InputForm />
-      <AmortisationChart />
+      <InputForm onSubmit={onSubmit} />
+      {loan && <AmortisationChart loan={loan} />}
```
Comment (achoi0330) [inline amortisation-calculator/src/App.js:20]: What is `loan &&` for?  is it to hide the chart before the input form's submitted?

Comment (leeningyap) [reply amortisation-calculator/src/App.js:20]: Correct

File: amortisation-calculator/src/components/AmortisationChart.js (hunk 1 above)
Comment (leeningyap) [reply amortisation-calculator/src/components/AmortisationChart.js]: Renamed dataset name

File: amortisation-calculator/src/components/AmortisationChart.js [hunk 3]
```diff
@@ -37,7 +26,7 @@ const AmortisationChart = props => {
         position: 'absolute',
         margin: '2em 0',
         left: '50%',
-        transform: 'translate(-50%, 0%)'
+        transform: 'translate(-50%, 0%)',
```
Comment (achoi0330) [inline amortisation-calculator/src/components/AmortisationChart.js:29]: dun need this comma

**Pull request merged**